code/datums/elements/amputating_limbs.dm

```diff
@@ -52,6 +55,9 @@
 		surgeon.balloon_alert(surgeon, "already busy!")
 		return COMPONENT_CANCEL_ATTACK_CHAIN
 
+	if(pre_hit_callback && pre_hit_callback.Invoke(victim) == FALSE)
```

Comment:
```suggestion
	if(pre_hit_callback && !pre_hit_callback.Invoke(victim))
```
Unless you are differentiating `FALSE` from other falsy values explicitly for some reason